In [9]:
H = 1 / 2
T = 1 / 2

print(H * T * T * H)

0.0625


1. Calculate the probability of flipping a balanced coin four times and getting each pattern: HTTH, HHHH and TTHH. 

Answer: The likelihood of flipping a coin and getting the patterns HTTH, HHHH, and TTHH is 6.25%.

2. If a list of people has 24 women and 21 men, then the probability of choosing a man from the list is 21/45. What is the probability of not choosing a man? 

Answer: 24/45

In [11]:
(.1) * (.00005)

5e-06

3. The probability that Bernice will travel by plane sometime in the next year is 10%. The probability of a plane crash at any time is .005%. What is the probability that Bernice will be in a plane crash sometime in the next year?

Answer: 5e-06

4. A data scientist wants to study the behavior of users on the company website. Each time a user clicks on a link on the website, there is a 5% chance that the user will be asked to complete a short survey about their behavior on the website. The data scientist uses the survey data to conclude that, on average, users spend 15 minutes surfing the company website before moving on to other things. What is wrong with this conclusion?

Answer: The conclusion is wrong because there is sample bias involved. Those who are offered the survey are not representative of the population because they are the only ones CLICKING on links. This leaves out an entire (likely large) subset of the population that may click on no links whatsoever...spending very little time on the company website.